### DISCUSSION TITLE
stop multiple submission

### DISCUSSION THREAD

**Discussion by akshayharshan** [Budibase/budibase#8332] [Help]
when submitting a form if we click the save button multiple times multiple entries of the same data happen, how to avoid this?

**Comment by agruenberg** [comment]
You can clear the data entry form, after submitting the data. 
Otherwise you could change the state of a state-variable and make a condition that controls the visibility of the save button.

**Reply by akshayharshan** [reply]
![Screenshot (189)](https://user-images.githubusercontent.com/79188989/196949356-316c7f35-787e-4cc4-a6c5-ec5f62d08494.png)

hope this is what you mentioned above. right?

**Reply by agruenberg** [reply]
I would suggest the order 1 -> 4 -> 2 -> 3

**Reply by akshayharshan** [reply]
@agruenberg thankyou.

**Reply by agruenberg** [reply]
Did it help?

**Reply by akshayharshan** [reply]
@agruenberg I tried the order that you suggested but then also I got multiple entries, so I tried 1->2->4->3 surprisingly it got worked.
